app.js

```diff
@@ -0,0 +1,48 @@
+// starts the express server
+// import other stuff from the folders
+
+const express = require('express');
+
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const cacheControl = require('express-cache-controller');
+const { createPool } = require('mariadb');
+const config = require('./api/config');
+const routes = require('./api/routes');
+
+
+
+
+const app = express();
+const port = process.env.NODE_ENV === 'development' ? '3005' : '3004';
+
+// database
+app.locals.pool = createPool(config.mysql);
+app.locals.pool.add('local', config.mysql0);
+// app.locals.pool.add('db1', config.mysql1);
+// app.locals.pool.add('db2', config.mysql2);
+// app.locals.pool.add('db3', config.mysql3);
```

Comment:
remove the pooling for now. we can come back to that.

Reply:
The app doesnt seem to run without the first line